user/tests/wiring/sleep20/sleep20.cpp

```diff
@@ -661,5 +667,139 @@ test(26_System_Sleep_With_Configuration_Object_Ultra_Low_Power_Mode_Wakeup_By_Wi
     assertEqual(result.error(), SYSTEM_ERROR_NONE);
     assertEqual((int)result.wakeupReason(), (int)SystemSleepWakeupReason::BY_NETWORK);
 }
+#endif
+#endif // HAL_PLATFORM_WIFI
+
+#if HAL_PLATFORM_CELLULAR
+test(23_System_Sleep_With_Configuration_Object_Stop_Mode_Execution_Time) {
+    constexpr uint32_t SLEEP_DURATION_MS = 1000;
+    Serial.printf("    >> Device enters stop mode. Please reconnect serial after %ld ms\r\n", SLEEP_DURATION_MS);
+    Serial.println("    >> Press any key now");
+    while(Serial.available() <= 0);
+    while (Serial.available() > 0) {
+        (void)Serial.read();
+    }
+
+    Serial.println("    >> Connecting to the cloud");
+    Cellular.on();
```

Comment:
Let's generalize this and use `Network` here, so that this can be verified on WiFi devices as well, within a single test.

Reply:
Done.